build.sbt

```diff
@@ -79,7 +79,7 @@ val sharedSettings = scalariformSettings ++  Seq(
 
   scalacOptions ++= {
     if (scalaVersion.value startsWith "2.12")
-      Seq("-opt:l:classpath")
```

Comment:
there is an option for inlining from the sources. Can we change to that?
How about:
https://gist.github.com/retronym/29932ea76712ff374d1363c9eda6eabe

`-optimize -opt-inline-from:com.twitter.algebird.*` or something, so we can inline from the current project?

Reply:
There is an option ```-opt:l:project``` which seems like the one that can be used, however it requires some research. For basic testing, it looks like it doesn't break binary compatibility.

Reply:
`-opt:l:project` is also deprecated in 2.12.3, I recommend upgrading to 2.12.3 and using `-opt:l:inline` with `-opt-inline-from:com.twitter.algebird.**`